engine/configurator/conf_yaml.c

```diff
@@ -28,21 +27,128 @@
 #define YAML_NODE_LINE_COLUMN(_n)   \
     (_n)->start_mark.line + 1, (_n)->start_mark.column + 1
 
-#define YAML_TARGET_CONTEXT_INIT \
-    { .oid = NULL, \
-      .value = NULL, \
-      .access = NULL, \
-      .type = NULL, \
-      .xmlvolatile = NULL, \
-      .substitution = NULL, \
-      .unit = NULL, \
-      .deps = SLIST_HEAD_INITIALIZER(deps), \
-      .cond = TRUE }
+typedef enum cs_yaml_node_type {
+    CS_YAML_NODE_TYPE_COMMENT,
+    CS_YAML_NODE_TYPE_INCLUDE,
+    CS_YAML_NODE_TYPE_COND,
+    CS_YAML_NODE_TYPE_REGISTER,
+    CS_YAML_NODE_TYPE_UNREGISTER,
+    CS_YAML_NODE_TYPE_ADD,
+    CS_YAML_NODE_TYPE_GET,
+    CS_YAML_NODE_TYPE_DELETE,
+    CS_YAML_NODE_TYPE_COPY,
+    CS_YAML_NODE_TYPE_SET,
+    CS_YAML_NODE_TYPE_REBOOT,/* Should it be added? */
+} cs_yaml_node_type;
+
+const te_enum_map cs_yaml_node_type_mapping[] = {
```

Comment:
`static` ?

Reply:
I've realized that I need to make all this stuff to be static. I will do it later. On weekend.

Reply:
Fixed